pepper-apis/src/main/java/org/broadinstitute/ddp/util/ResponseUtil.java

```diff
@@ -27,6 +29,16 @@ public static void halt422ErrorResponse(Response res, String errorCode) {
 
     }
 
+    public static String renderPageNotFound(Response response) {
+        response.type(ContentType.APPLICATION_JSON.getMimeType());
+        ApiError apiError = new ApiError(ErrorCodes.NOT_FOUND, "This page was not found.");
+        return transformer.render(apiError);
+    }
+
+    public static HaltException halt404PageNotFound(Response response) {
+        return halt(HttpStatus.SC_NOT_FOUND, renderPageNotFound(response));
+    }
+
```

Comment:
Refactored stuff a bit to try to maximize DRYness and to make sure our Page Not Found errors look the same everywhere.